### ISSUE TITLE
Feature Request: Add Tick Box to account creation process to confirm key saved before enable next action

### ISSUE THREAD

**Issue by roger-eosuk** [Telos-Foundation/Sqrl#95]
Hi Marlon

I think I have posted this before in Slack but I don't think you ever saw it, it was a great idea! 
Like this http://tinyurl.com/y5yyro69 

Actually here is the original http://tinyurl.com/y6tch93t 
Add a tick box and a red message that has to be confirmed, before the "Keys Are in a Safe Place" button is enabled.

I am not sure if this is easily doable or not, what do you think?

Cheers

Rog

**Comment by marlonwilliams** [comment]
Hi Roger,

The "Keys are in a safe place" button is only enabled if the user clicks the "Copy keys/already have keys" button. This action indicates that the user knowingly selected that option before being able to move forward. I think one important metric to gauge whether the existing method works is the number of users who reported losing their keys or access to their account as a result of this. If I'm not mistaken, there hasn't been any or, at most, one that I remember that may be vaguely related to this.

In summary, I see this button as somewhat redundant and unnecessary based on the past 6 months of Sqrl usage. Happy to hear the thoughts of others on this thread!

**Comment by roger-eosuk** [comment]
Hi Marlon

 

Funnily enough I just stumbled across something on the ScotCoin wallet very similar

 



 

See how the default click-click-clickers will have to stop before and read and then check the tick box before they can lose their keys forever?

I was meaning a flow like this in Sqrl would be really helpful to very new users.

 

Best regards

 

Roger

 

--Roger Davies

Advanta Productions Limited

Web Apps & Internet Solutions

https://advantapro.com

Tel 0115 822 4546

**Comment by marlonwilliams** [comment]
Did you attached any images? Not seeing the flow you're referring to. Would love to take a look.

**Comment by roger-eosuk** [comment]
It did not come through must have been an email reply, but they basically have done as i suggested in the screenshots linked at the top of the page, so that the DEFAULT action of just clicking, results in a pause, not losing your keys. 

This behavior is also what Windows does when you try to install a program it doesn't know of yet(like Sqrl!) for those people who just click without reading, they get stuck and have to stop and read, and then click More Info, and click Run Anyway.

It is this kind of flow, where a loop back is required in order to continue, that is what I was suggesting for Sqrl.
